frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -434,9 +452,13 @@ function setupLayerForVolumeTracing(
   const fallbackLayer = layers[fallbackLayerIndex];
   const boundaries = getBoundaries(dataset);
 
+  // We do not adjust the elementClass here if it is uint64, because uint64 volume tracings are not supported.
+  // Not adjusting the elementClass will cause the later check for supported layer element classes to fail
+  // if the segmentation of type uint64.
   const tracingLayer = {
     name: tracing.id,
     elementClass: tracing.elementClass,
+    originalElementClass: tracing.elementClass,
```

Comment:
Does the `originalElementClass` need to be set here at all? In the type definition it's declared as optional.

Reply:
Thanks, you are correct. I'll revert the changes here :+1: